package.json

```diff
@@ -3,9 +3,10 @@
   "homepage": "http://postcss.com/",
   "scripts": {
     "start": "babel-node scripts/build --server --dev --open",
-    "static": "babel-node scripts/build --static --production",
+    "static": "npm run lint && babel-node scripts/build --static --production",
```

Comment:
Deploy use "npm test" (which use "npm static" so that the same.
Anyway lintin mg is already done once during the build itself, via webpack and eslint-loader.